### PR TITLE
Fledge specific documentation updates for v2.6.0

### PR DESCRIPTION
**Pull request by ashish-jabble**

### CODE REVIEW

**Review by MarkRiddoch — APPROVED** [review]

File: docs/91_version_history.rst [hunk 1]
```diff
@@ -33,107 +33,73 @@ Release Date: 2024-10-24
...
+       - The ordering of the tabs in the configuration user interface is no longer alphabetical but controlled by the configuration category itself.
        - The user interface now hides configuration options the user does not have permission to change.
-       - The rendering of the developer menu has been bough into line with other sub-menu rendering.
+       - The rendering of the developer menu has been brought into line with other sub-menu rendering.
 
 
     - Bug Fix:
 
-       - An issue that causes the sidebar to fail to open when connecting to a FogLAMP User Interface from FogLAMP Manage has been resolved.
        - An issue with the user interface incorrectly displaying a timestamp has been resolved.
```
[the remark this answers is not in the document]
Comment (ashish-jabble) [reply docs/91_version_history.rst:76]: throughout where we display time

File: docs/91_version_history.rst [hunk 2]
```diff
@@ -33,107 +33,73 @@ Release Date: 2024-10-24
...
+       - fledge-south-s2opcua now supports for the OPC UA Data Change Filter has been added. This filter type is defined in the `OPC UA Specification, Part 4, Section 7.22.2 <https://reference.opcfoundation.org/Core/Part4/v105/docs/7.22.2>`_. The Data Change Filter allows OPC UA clients (such as this plugin) to request that the OPC UA server send data value updates only if the server's data values have changed significantly. With careful tuning, you can reduce the data traffic from OPC UA server to the client without significant loss of fidelity. This plugin has been upgraded to use Systerel's `S2OPC Toolkit Version 1.5.0 <https://gitlab.com/systerel/S2OPC/-/releases/S2OPC_Toolkit_1.5.0>`_.
```
[the remark this answers is not in the document]
Comment (ashish-jabble) [reply docs/91_version_history.rst:94]: See https://reference.opcfoundation.org/Core/Part4/v105/docs/7.22.2

File: docs/91_version_history.rst [hunk 3]
```diff
@@ -33,107 +33,74 @@ Release Date: 2024-10-24
 
     - New Features:
 
-       - A new section relating to all aspects of monitoring within Fledge has been added to the documentation
-       - Monitoring of the available space on the disk that holds the SQLite buffer for readings ingested by Fledge has been added. Entries will be written to the error log to predict the expiration of the disk space and also logs when the spaces left available falls below 10% and 5%.
-       - Failures to send data north via the north service raises an alert, this is now cleared if the flow of data north later resumes.
+       - Monitoring of the available space on the disk that holds the SQLite buffer for readings ingested by Fledge has been added. Entries will be written to the error log to predict the expiration of the disk space and also logs when the spaces left available fall below 10% and 5%.
+       - Failures to send data north via the north service raise an alert, this is now cleared if the flow of data north later resumes.
```
Comment (RVerhoeff) [inline docs/91_version_history.rst:37]: Start a new sentence at "This is now cleared if..."

**Review by RVerhoeff — CHANGES_REQUESTED** [review]
One minor sentence fix. I am on vacation this week so please ask one of the other reviewers to finish this.

File: docs/91_version_history.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (praveen-garg) [reply docs/91_version_history.rst:95]: I mean it should be 

now supports


The words "has been added" should be removed.

File: docs/91_version_history.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (praveen-garg) [reply docs/91_version_history.rst:76]: I think in both cases, the API and UI, it was fixed on Backup & Restore only.

File: docs/91_version_history.rst [hunk 4]
```diff
@@ -33,107 +33,73 @@ Release Date: 2024-10-24
...
-
+       - fledge-filter-asset now supports to allow regulator expressions to be used in the asset name to match when applying the filter.
+       - fledge-filter-delta has been enhanced to give greater control over the action when one or more datapoints in an asset are detected as changing.
+       - fledge-notify-operation now supports data substitution as with the other control-related notification delivery plugins.
+       - fledge-north-OMF *action* code in the HTTP header is normally set to *update* for OMF Data messages. This causes old values to be updated if the timestamps match and new data to be properly compressed by the PI Data Archive. If the AVEVA PI Buffer Subsystem is configured, however, the *update* action code is converted to an internal PI storage mode that causes new data to bypass compression, which can result in unnecessarily storing too many data values. To work around this issue, the OMF North plugin now allows you to change the OMF Data action code to *create*. This should only be done when the AVEVA PI Buffer Subsystem is configured.
```
[the remark this answers is not in the document]
Comment (praveen-garg) [reply docs/91_version_history.rst]: The OMF north plugin's *action* code in the HTTP header is typically set to *update* for OMF Data messages. This setting allows old values to be updated when timestamps match and ensures that new data is compressed correctly by the PI Data Archive. However, if the AVEVA PI Buffer Subsystem is configured, the *update* action code is converted to an internal PI storage mode, which causes new data to bypass compression and may lead to excessive storage of too many data values. To address this issue, the OMF North plugin now allows you to change the OMF Data action code to *create*. This option should only be used when the AVEVA PI Buffer Subsystem is configured.

**Pull request merged**